filebeat/docs/multiline.asciidoc

```diff
@@ -17,7 +17,7 @@ Also read <<yaml-tips>> and <<regexp-support>> to avoid common mistakes.
 [[multiline]]
 === Configuration options
 
-You can specify the following options in the `filebeat.prospectors` section of
+You can specify the following options in the `filebeat.inputs` section of
```

Comment:
Should we use asciidoc variable here? {beatname_uc}

Reply:
yup. I've fixed the references in this file, too. I'm doing these changes in small batches with other changes because it's not always a simple text replacement. Formatting affects how the attributes are interpreted.